.github/workflows/deploy-preview.yml

```
@@ -55,13 +55,19 @@ jobs:
           name: dist-files
           path: dist
 
+      - name: Set short git commit SHA
+        id: vars
+        run: |
+          calculatedSha=$(git rev-parse --short ${{ github.event.pull_request.head.sha }})
+          echo "COMMIT_SHORT_SHA=$calculatedSha" >> $GITHUB_ENV
+    
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)